### PR TITLE
VDB-1020 - 2.16 release

### PR DESCRIPTION
**Pull request by elizabethengelman**
- note: `MCD_FLIP_ETH_B`, `MCD_FLIP_ETH_C`, `MCD_FLIP_REP_A`, `MCD_FLIP_ZRX_A`, `MCD_FLIP_OMG_A`, `MCD_FLIP_DGD_A` and `MCD_FLIP_GNT_A` weren't included in the addresses for 2.16, so I remove them from the regular and docker config files, but left them in `testing.toml` as a way to document what we have. I'd be happy to remove them from there as well if folks thing it's cleaner.

### CODE REVIEW

File: environments/docker.toml [hunk 1]
```diff
@@ -423,81 +367,51 @@
...
...e,"internalType":"uint256","name":"id","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"lot","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"bid","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"tab","type":"uint256"},{"indexed":true,"internalType":"address","name":"usr","type":"address"},{"indexed":true,"internalType":"address","name":"gal","type":"address"}],"name":"Kick","type":"event"},{"anonymous":true,"inputs":[{"indexed":true,"internalType":"bytes4","name":"sig","type":"bytes4"},{"indexed":true,"internalType":"address","name":"usr","type":"address"},{"indexed":true,"internalType":"bytes32","name":"arg1","type":"bytes32"},{"indexed":true,"internalType":"bytes32","name":"arg2","type":"bytes32"},{"indexed":false,"internalType":"bytes","name":"data","type":"bytes"}],"name":"LogNote","type":"event"},{"constant":true,"inputs":[],"name":"beg","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[{"internalType":"uint256","name":"","type":"uint256"}],"name":"bids","outputs":[{"internalType":"uint256","name":"bid","type":"uint256
```
Comment (rmulhol) [inline environments/docker.toml]: Just verifying - did you paste in the new ABIs? There were some changes (at least to formatting) for the mainnet deploy, but idk if that might just be differences in how it was compiled

File: transformers/integration_tests/bite.go [hunk 2]
```diff
@@ -40,9 +40,8 @@ var _ = Describe("Bite Transformer", func() {
 		Topic:             constants.BiteSignature(),
 	}
 
-	// TODO: replace block number when there is an updated Cat bite event
-	XIt("fetches and transforms a Bite event from Kovan chain", func() {
-		blockNumber := int64(8956422)
+	It("fetches and transforms a Bite event from Kovan chain", func() {
```
Comment (rmulhol) [inline transformers/integration_tests/bite.go]: 🎉

File: environments/docker.toml (hunk 1 above)
Comment (elizabethengelman) [reply environments/docker.toml]: Yep, I got the ABIs from the [2.16 changelog](https://changelog.makerdao.com/releases/kovan/0.2.16/abi/index.html).

Comment (elizabethengelman) [reply environments/docker.toml]: Yep, I got the ABIs from the 2.16 changelog.

**Pull request merged**